### PR TITLE
feat(languages): add Gujarati language support (prince-patel23)

### PR DESCRIPTION
**Pull request by prince-patel23**
### Description

This PR introduces support for the Gujarati language in Monkeytype. Users can now practice typing in Gujarati.

### Checks

- [x] Adding a language?
  - [x] Edited `_list.json`, `_groups.json`, and added `languages.json`.

- [x] Checked for related open issues.
- [x] PR title follows the Conventional Commits standard. ([Conventional Commits](https://www.conventionalcommits.org) for more info)
- [x] GitHub username included in parentheses at the end of the PR title.

Closes #issue-number (if applicable)

### CODE REVIEW

**Comment by prince-patel23** [comment]
Hi @Miodec ,👋

**_I've implemented the suggested change by adding the "noLazyMode": true property in the Gujarati.json file. All checks are completed,_** and I believe this change will enhance the Gujarati language support.

Could you please review and consider merging the changes? I appreciate your time!

Thanks!
@prince-patel23

File: frontend/static/languages/gujarati.json [hunk 1]
```diff
@@ -0,0 +1,223 @@
+{
+    "name": "gujarati",
+    "rightToLeft": false,
+    "ligatures": true,
```
[the remark this answers is not in the document]
Comment (prince-patel23) [reply frontend/static/languages/gujarati.json:4]: **given suggestion added.**

**Review by unrealapex — APPROVED** [review]

**Comment by unrealapex** [comment]
Please don't @ people unless you *specifically need their attention*. GitHub automatically notifies people.

**Comment by Miodec** [comment]
Thanks. Be careful about mentioning people for no reason ;)

**Pull request merged**